pennylane/pulse/rydberg.py

```diff
@@ -54,10 +54,11 @@ def rydberg_interaction(
         wires (list): List of wires containing the wire values for all the atoms. This list should
             have the same length as ``register``. If ``None``, each atom's wire value will
             correspond to its index in the ``register`` list.
-        interaction_coeff (float): Rydberg interaction constant in units: :math:`\text{MHz} \times \mu \text{m}^6`.
-            Defaults to :math:`862690 \text{ MHz} \times \mu \text{m}^6`. This value is based on an assumption that
-            frequencies and energies in the Hamiltonian are provided in units of MHz.
-        max_distance (float): Threshold for distance in :math:`\mu \text{m}` between two Rydberg atoms beyond which their
+        interaction_coeff (float): Defaults to :math:`862690 \times 2 \pi \text{MHz } \mu\text{m}^6`.
```

Comment:
```suggestion
        interaction_coeff (float): Defaults to :math:`862690 \times \text{MHz } \mu\text{m}^6`.
```
I'm not entirely sure, but that is how I interpret the text.

Reply:
I would opt to be explicit about $2\pi$ factors

Reply:
Added alternative suggestion below

Reply:
It would be the only place in the doc string where it is not explicit, and also is in conflict with the [bloqade docs](https://queracomputing.github.io/Bloqade.jl/dev/hamiltonians/) so I left it with the version as is, which I quite like

![image](https://user-images.githubusercontent.com/43949391/235345439-6512b166-d8e7-4d46-9770-b01ce323abdf.png)